services/movie.py

```diff
@@ -1,11 +1,12 @@
 from django.db.models import QuerySet
-
 from db.models import Movie
+from django.db import transaction
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None,
```

Comment:
If argument can be `None` you should show in in type hint. Fix everywhere, please
```suggestion
    title: list[str] | None = None,
```